autoload/tsuquyomi.vim

```diff
@@ -94,6 +94,21 @@ function! s:writeToPreview(content)
   setlocal nomodifiable readonly
   silent wincmd p
 endfunction
+
+function! s:setqflist(quickfix_list, ...)
+  " 0: Do not close cwindow automatically
+  " 1: Close cwindow automatically
+  let auto_close = len(a:000) ? a:0 : 0
```

Comment:
Notify errors to user should do silent.
If Hooking TsuAsyncGeterr to Texchanged autocmd, Open / Close cwindow many times.
This is annoying.